bin/kafka-producer-perf-test.sh

```diff
@@ -17,4 +17,4 @@
 if [ "x$KAFKA_HEAP_OPTS" = "x" ]; then
     export KAFKA_HEAP_OPTS="-Xmx512M"
 fi
-exec $(dirname $0)/kafka-run-class.sh kafka.tools.ProducerPerformance $@
+exec $(dirname $0)/kafka-run-class.sh org.apache.kafka.clients.tools.ProducerPerformance $@
```

Comment:
Yeah, that's the thing that was especially weird for me, the fact that it was very different to every other tool we had (as you say, because it doesn't use `jopt`). Is it worth filing an issue to use `jopt` so that it works similarly to other command-line tools?